css/wsuwp-plugin-idonate-src.less

```diff
@@ -660,6 +660,15 @@
 	.ui-menu-item.ui-state-focus {
 		color: #981e32;
 	}
+	li.narrow-search {
+		font-style: italic;
+		padding-top: 10px;
+		text-transform: lowercase;
+		border-top: 1px #8d959a solid;
```

Comment:
If you add `pointer-events:none;`, this will disable the message from being clicked until `disabled` can be applied.